Tools/CISupport/ews-build/steps_unittest.py

```diff
@@ -214,6 +214,26 @@ def behavior(command):
     return behavior
 
 
+class TestGitHub(unittest.TestCase):
+    def test_pr_url(self):
+        self.assertEqual(
+            GitHub.pr_url(1234),
+            'https://github.com/WebKit/WebKit/pull/1234',
+        )
+
+    def test_commit_url(self):
```

Comment:
Thanks for adding these. Can also add unit-tests for cases when repository_url was provided (valid and invalid).